CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
 cmake_minimum_required(VERSION 3.31)
 project(betafold)
 
-set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD 23)
```

Comment:
I copy/pasted the CMakeLists.txt file from another project I own and from which i grant permission for this project to use the copy/pasted code. The edits in this file:

  - change the C++ standard to 23 from 20
  - Change the prefix/path name from `cbetafold.betafold` to just `betafold`
  - Enable us to build/run a `test` executable defined in `main.cpp` so we can fiddle with our library and show off examples if we like by fleshing out `main.cpp`